decimal64.go

```diff
@@ -454,9 +453,36 @@ func (d Decimal64) IsInt() bool {
 		return false
 	}
 }
-func (d Decimal64) isZero() bool {
+
+// IsSubnormal returns true iff d is a subnormal.
+func (d Decimal64) IsSubnormal() bool {
 	fl, _, _, significand := d.parts()
-	return significand == 0 && fl == flNormal
+	return significand != 0 && significand < decimal64Base && fl == flNormal
+}
+
+// updateMag updates the magnitude of the dec object
```

Comment:
updatemag -> isZero
Sorry this must've been one of my typos

Reply:
or remove this comment alltogether; nonexported identifier

Reply:
yea i removed the comment :)